src/internal/m365/service/sharepoint/backup.go

```diff
@@ -267,13 +218,63 @@ func collectPages(
 			el.AddRecoverable(ctx, clues.Wrap(err, "creating page collection path").WithClues(ctx))
 		}
 
-		collection := NewCollection(
+		collection := site.NewCollection(
+			dir,
+			ac,
+			site.Pages,
+			su,
+			bpc.Options)
+		collection.SetBetaService(betaService)
+		collection.AddJob(tuple.ID)
+
+		spcs = append(spcs, collection)
+	}
+
+	return spcs, el.Failure()
+}
+
+func collectLists(
```

Comment:
I'm still debating where `collectPages`, `collectLists` and `collectLibraries` should be. Any thoughts?

Reply:
Probably in `/m365/collections/site`.